views/agreement.hbs

```diff
@@ -217,7 +324,24 @@ input[type="submit"] {
    <h4 id="signature-individual">If signing as an <a href="#individual">Individual</a></h4>
   </summary>
   <div>
-   <p>I represent that I do not <a href="https://whatwg.org/ipr-policy#2101-work-in-the-field-of-web-technologies">work in the field of web technologies</a> as an employee, contractor, or agent of another person or legal entity, and make this <a href="#agreement">Agreement</a> in my personal capacity only.</p>
+   <p>Select one:
+
+   <div id="individual-signature-selection">
+    <label>
+     <input type="radio" name="individual_type" value="self" checked>
+     I represent that I do not
+     <a href="https://whatwg.org/ipr-policy#2101-work-in-the-field-of-web-technologies">work in the field of web technologies</a>
+     as an employee, contractor, or agent of another person or legal entity, and make this
+     <a href="#agreement">Agreement</a> in my personal capacity only.
+    </label>
+
+    <label>
+     <input type="radio" name="individual_type" value="invited">
+     I have been invited by the
+     <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a> and I agree that I am
+     entering into this <a href="#agreement">Agreement</a> in my personal capacity only.
+    </label>
+   </div>
```

Comment:
If JavaScript is enabled we should do this automatically based on the Scope choice.

Reply:
That might not be ideal: (1) per one of my above comments, I'd rather have them select scopes like normal; (2) per the other of my above comments, if we can make this an affirmative choice, it might decrease the number of mistaken individual signatures.

Reply:
+1 on the affirmative choice--and I think this was particularly important to counsel.